src/components/EmojiPicker/EmojiSkinToneList.tsx

```diff
@@ -39,7 +41,7 @@ function EmojiSkinToneList() {
         // eslint-disable-next-line react-hooks/exhaustive-deps -- only run when preferredSkinTone updates
     }, [preferredSkinTone]);
 
-    const currentSkinTone = getSkinToneEmojiFromIndex(preferredSkinTone);
+    const currentSkinTone = getSkinToneEmojiFromIndex(preferredSkinTone as number);
```

Comment:
I think best will be to adjust types of those parameters instead of asserting it to number